source/coll_crystal.f90

```diff
@@ -244,6 +244,8 @@ subroutine cry_doCrystal(ie,iturn,j,mat,x,xp,z,zp,s,p,x0,xp0,zlm,s_imp,nhit,nabs
   real(kind=fPrec) xp_temp,xp_shift,xp_rot,xp_int,xp_tangent
   real(kind=fPrec) tilt_int,shift,delta,a_eq,b_eq,c_eq
 
+  logical isImp
```

Comment:
Could you maybe add here the `intent` statement for this var? It helps clarifying what's the role of the variable...
You can find examples in other modules.

Reply:
Yes, all function variables should have the intent statement.

Reply:
Thanks for the remark, the `intent` statement has been added in one of the latest commits.